relay-server/src/actors/envelopes.rs

```diff
@@ -1612,13 +1521,14 @@ impl EnvelopeProcessor {
         &self,
         state: &mut ProcessEnvelopeState,
     ) -> Result<(), ProcessingError> {
-        if !state.project_state.has_feature(Feature::MetricsExtraction) {
-            return Ok(());
-        }
+        let config = match state.project_state.config.transaction_metrics {
+            Some(ErrorBoundary::Ok(ref config)) => config,
+            None | Some(ErrorBoundary::Err(_)) => return Ok(()),
```

Comment:
nit: Is this equivalent to `_ => return Ok(())`?

Reply:
yes, i have the tendency to avoid `_` but it's not healthy in this case